packages/eas-cli/src/graphql/types/credentials/AppleDistributionCertificate.ts

```diff
@@ -0,0 +1,28 @@
+import { Fragment } from '../../fragment';
+import { AppleTeam } from './AppleTeam';
+
+export interface AppleDistributionCertificate {
+  id: string;
+  certificateP12: string;
+  certificatePassword: string;
+  serialNumber: string;
+  developerPortalIdentifier: string;
+  validityNotBefore: Date;
+  validityNotAfter: Date;
+  appleTeam?: AppleTeam;
+}
+
+export const AppleDistributionCertificateFragment: Fragment = {
+  name: 'appleDistCert',
+  definition: `
+    fragment appleDistCert on AppleDistributionCertificate {
+      id
+      certificateP12
+      certificatePassword
+      serialNumber
+      developerPortalIdentifier
+      validityNotBefore
+      validityNotAfter
```

Comment:
is `AppleDistributionCertificate` supposed to be the type of `AppleDistributionCertificateFragment`?  if so, should probably resolve apple team in the fragment too, for example: 
```
    appleTeam {
      id
      teamIdentifier
      ...
    }
```